src/lib/parsers/include/sol-json.h

```diff
@@ -698,19 +697,19 @@ char *sol_json_escape_string(const char *str, char *buf, size_t len) SOL_ATTR_NO
  */
 #define SOL_JSON_ESCAPE_STRING(s) ({ \
         size_t str_len ## __COUNT__  = sol_json_calculate_escaped_string_len(s); \
-        sol_json_escape_string(s, (char *)alloca(str_len ## __COUNT__), str_len ## __COUNT__); \
+        SOL_BUFFER_DECLARE_STATIC(aux_buf ## __COUNT__, str_len ## __COUNT__); \
```

Comment:
before it used explicit call to `alloca()`, now we rely on newer C99 (?) standard to allocate stuff in the stack using a variable. No big deal, but we should keep an eye if this will work in other compilers.

@lpereira